libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextSelectors.ts

```diff
@@ -102,7 +128,12 @@ export const selectAttributeFilterDisplayForms = createSelector(selectSelf, (fil
 /**
  * Selects map of display form metadata objects referenced by attribute filters.
  *
- * @alpha
+ * @remarks
+ * Invocations before initialization lead to invariant errors.
+ *
+ * @returns a {@link ObjRefMap} of {@link @gooddata/sdk-backend-spi#IAttributeDisplayFormMetadataObject}
+ *
+ * @public
  */
 export const selectAttributeFilterDisplayFormsMap = createSelector(selectSelf, (filterContextState) => {
```

Comment:
This should probably stay not public, the ObjRefMap it uses internally is not public yet.